src/pages/EditCardPage/CardItems.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+import Button from '../../components/UI/Button/Button'
+import classes from './CardItems.module.css'
+import { useHistory } from 'react-router-dom'
+
+const CardItems = () => {
+    const history = useHistory()
+
+    const dummy_card = [
+        {
+            id: 1,
+            front: '1+1',
+            back: '2',
+        },
+        {
+            id: 2,
+            front: 'Apple',
+            back: 'ringo',
+        },
+        {
+            id: 3,
+            front: 'something asdf sadf ',
+            back: 'wersaf asdf ',
+        },
+        {
+            id: 4,
+            front: 'qwr asf wqr ',
+            back: 'asfd qr saf ',
+        },
+        {
+            id: 5,
+            front: '1+1',
+            back: 'wqers asf asf wqers asf asf wqers asf asf wqers asf asfwqers asf asf wqers asf asfwqers asf asf wqers asf asf f asf wqers asf asfwqers asf asf wqers asf asff asf wqers asf asfwqers asf asf wqers asf asff asf wqers asf asfwqers asf asf wqers asf asff asf wqers asf asfwqers asf asf wqers asf asf ',
+        },
+        {
+            id: 6,
+            front: 'wqers asf asf wqers asf asf wqers asf asf wqers asf asfwqers asf asf wqers asf asfwqers asf asf wqers asf asf f asf wqers asf asfwqers asf asf wqers asf asff asf wqers asf asfwqers asf asf wqers asf asff asf wqers asf asfwqers asf asf wqers asf asff asf wqers asf asfwqers asf asf wqers asf asf ',
+            back: 'rwerwesf asf qwr saf ',
+        },
+        {
+            id: 7,
+            front: 'wqers asf safwqr qwer  ',
+            back: 'rwerwesf asf qwr saf ',
+        },
+    ]
+
+    const onClickEditHandler = (id) => {
+        console.log('clicked edit button')
+        console.log(`Id is ${id}`)
+        history.push(`/add-card?${id}`)
```

Comment:
Works great we should consider having /add-card page route back to /edit-card if cancel or save are clicked

Reply:
Thanks, I will change these routes in the story branch.